graphql_compiler/exceptions.py

```diff
@@ -36,4 +36,4 @@ class GraphQLInvalidArgumentError(GraphQLError):
 
 
 class GraphQLPaginationError(GraphQLError):
-    """Exception raised when the pagination planner was unable to find a plan."""
+    """Exception raised when the pagination planner was unable to find a good plan."""
```

Comment:
Goodness is a bit ambiguous imo; shouldn't we only raise an exception if something is truly broken?

Reply:
Good catch, I didn't intend to leave this in the PR.